src/modules/editor/SoundEditor/MusicManager.ts

```diff
@@ -0,0 +1,44 @@
+
+import * as Tone from "tone";
+import { Time } from "tone/build/esm/core/type/Units";
+
+const AllNotes = ["C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B"];
+
+interface SynthNote {
+  note: string;
+  duration: string;
+  type: SynthType;
+  volume: number;
+}
+
+export type { SynthNote };
+
+enum SynthType {
+  SYNTH = 0,
+  DUOSYNTH = 1,
+  POLYSYNTH = 2,
+  // NOISE = 3
+}
+
+class MusicManager {
+  public numberToNote(number: number): string {
+    if (number < 0) {
+      throw new Error("Number out of bounds");
+    }
+    const note = AllNotes.length - number % AllNotes.length - 1;
```

Comment:
This looks ambiguous to me -- I think adding parenthesis to part of the expression helps with readability